backend/corpora/upload_success/app.py

```
@@ -17,9 +18,7 @@ def success_handler(event: dict, context) -> None:
     """
     dataset_uuid = event["dataset_uuid"]
 
-    try:
-        with db_session_manager() as session:
-            success_status = {DbDatasetProcessingStatus.processing_status: ProcessingStatus.SUCCESS}
-            processing_status_updater(session, dataset_uuid, success_status)
-    except Exception:
```

Realized that I was swallowing all exceptions here, which is bad!